start.sh

```diff
@@ -35,13 +35,13 @@ chown -R backup:backup /data
 # Set up crontab.
 # CRONTAB is set in the Dockerfile
 # CRON_SCHEDULE and the other environment variables are set in docker-compose.yml
+echo "Setting up backup cronjob"
 echo "" > $CRONTAB
+echo "${BACKUP_SCHEDULE} DB_HOST=${DB_HOST} DB_NAME=${DB_NAME} DB_USER=${DB_USER} DB_PASS=${DB_PASS} KEEP_DAYS=${KEEP_DAYS} /app/backup.sh" >> $CRONTAB
 
-if [ "$BACKUP_CRON_ENABLE" = true ]; then
-    echo "Setting up backup cronjob"
-    echo "${BACKUP_SCHEDULE} DB_HOST=${DB_HOST} DB_NAME=${DB_NAME} DB_USER=${DB_USER} DB_PASS=${DB_PASS} KEEP_DAYS=${KEEP_DAYS} /app/backup.sh" >> $CRONTAB
-else
-    echo "Setting up do-nothing cronjob: automatic backups are disabled"
+if [ "$BACKUP_CRON_ENABLE" != true ]; then
```

Comment:
This should be moved to the beginning and in the inner part of the if case there should be an [exit statement](https://bash.cyberciti.biz/guide/Exit_command)

Reply:
Ok, this is not what I wanted to achieve though. my idea (see description above) was to start the cron deamon in both cases, but if backups are disabled, the rule is simply not written to the cron tab (and nothing is done). The sole reason of having an empty cron job is to keep the container running for easy manual testing. With your change the backup container immediately exits after creation. Do you prefer this?
If we keep the "dummy deamon", I'll rewrite this in a more concise and clear way, given the confusion. this should be really simple

Reply:
sorry, then https://github.com/MaRDI4NFDI/docker-backup/pull/17/commits/e8e86973296041bc3ce55a8de366b5c83c24e441 is the preferable solution, sorry. Can you merge the change in the original form?